OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -324,6 +325,10 @@ private void updateDataModel() {
 
     }
 
+    private void showPurchaseSuccessSnackbar() {
+        new SnackbarErrorNotification(pager2).showError(R.string.purchase_success_message);
+    }
+
```

Comment:
this method is not getting called, better to remove it.

Reply:
removed